lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -850,17 +850,12 @@ namespace Js
         this->weakReferenceDictionaryList.Prepend(this->GeneralAllocator(), weakReferenceDictionary);
     }
 
-    RecyclableObject *ScriptContext::GetMissingPropertyResult(Js::RecyclableObject *instance, Js::PropertyId id)
+    RecyclableObject *ScriptContext::GetMissingPropertyResult()
```

Comment:
This is good.
While you are here - could you check and remove unwanted params for GetNullPropertyResult and GetNullItemResult below?